script.sh

```diff
@@ -32,11 +32,16 @@ echo "::group::🐶 Installing reviewdog (${REVIEWDOG_VERSION}) ... https://gith
   curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b "${REVIEWDOG_PATH}" "${REVIEWDOG_VERSION}" 2>&1
 echo '::endgroup::'
 
-echo '::group:: Installing tfsec (latest) ... https://github.com/tfsec/tfsec'
+echo "::group:: Installing tfsec (${INPUT_TFSEC_VERSION}) ... https://github.com/tfsec/tfsec"
   test ! -d "${TFSEC_PATH}" && install -d "${TFSEC_PATH}"
 
+  if [[ "${INPUT_TFSEC_VERSION}" = "latest" ]]; then
```

Comment:
I was confused seeing `=` instead of `==` here, but turns out [there is no difference when using `[[ ]]`](https://stackoverflow.com/questions/12948456/is-there-any-difference-between-and-operators-in-bash-or-sh/12948608)!